### PR TITLE
Ticket 4/api integration

### PR DESCRIPTION
**Pull request by JoaoNacho-SG**
Added new API

### CODE REVIEW

File: src/components/Pages/Home/HeroSection/index.js [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 import style from "./herosection.module.scss";
 import { Layout } from "../../../layout/Layout";
-import { getSingleProduct } from "../../../../utils/api";
 import { Spinner } from "../../../general/Spinner";
 
-export const HeroSection = () => {
-  const [product, setProduct] = useState({});
-  const [loading, setLoading] = useState(true);
+export const HeroSection = ({ product }) => {
+  // const [loading, setLoading] = useState(true);
```
Comment (catlaranjeiro) [inline src/components/Pages/Home/HeroSection/index.js]: is this necessary?

File: src/components/views/Home/index.js [hunk 2]
```diff
@@ -10,16 +10,36 @@ import { BestIngredients } from "../../Pages/Home//BestIngredients";
 import { LearnMore } from "../../general/LearnMore";
 import { JoinSooth } from "../../general/JoinSooth";
 import { Footer } from "../../Footer";
+import { getFiftyProducts } from "../../../utils/api";
 
 export const Home = () => {
+  useEffect(() => {
+    (async () => {
+      const productsFromApi = await getFiftyProducts();
+      localStorage.setItem("products", JSON.stringify(productsFromApi.data));
```
Comment (catlaranjeiro) [inline src/components/views/Home/index.js]: Let's review this logic, I forgot to update the ticket requirements

**Review by catlaranjeiro — CHANGES_REQUESTED** [review]
Clean unecessary comments from code. We will review the localStorage, since I forgot to update that specific part of the ticket, but you filled all the requirements, so great work! 🚀

File: src/components/Pages/Home/HeroSection/index.js (hunk 1 above)
Comment (JoaoNacho-SG) [reply src/components/Pages/Home/HeroSection/index.js]: not really. I will remove the comment

**Review by catlaranjeiro — APPROVED** [review]
all good 👍

**Pull request merged**